sos/report/plugins/openshift.py

```diff
@@ -73,6 +80,10 @@ def _check_oc_function(self):
         """Check to see if we can run `oc` commands"""
         return self.exec_cmd('oc whoami')['status'] == 0
 
+    def _check_localhost_kubeconfig(self):
+        """Check if the localhost.kubeconfig exists with system:admin user"""
+        return self.path_exists(self.master_localhost_kubeconfig)
```

Comment:
This should instead be checking the `kubeconfig` option value. Otherwise, we're still gating on a static file even if a user uses a different config file.